### DISCUSSION TITLE
Can you implement Faker.js

### DISCUSSION THREAD

**Discussion by kevingrondin** [strapi/strapi#6372] [Legacy Topics]
Hello

During the creation of the database would it be possible to add false data, because it is very useful to make these first tests, **Faker.js** has many options for that.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by derrickmehaffy** [comment]
Please see the following example using faker: https://www.youtube.com/watch?v=DpR7NvJ2cAA

**Reply by kevingrondin** [reply]
Perfect it's exactly what y want
